grow/pods/podcache.py

```diff
@@ -41,12 +55,59 @@ def document_cache(self):
         """Cache for specific document properties."""
         return self._document_cache
 
+    @property
+    def object_cache(self):
+        """Global object cache."""
+        return self.get_object_cache(self.KEY_GLOBAL)
+
+    def create_object_cache(self, key, write_to_file=False, can_reset=True, values=None):
```

Comment:
I am treating the object cache as defaulting to a runtime only cache with the option to persist it to a file. The main idea was to keep the cache file from getting bloated with transient objects. Especially if the authors don't realize that their cache is being persisted. Also, having a large cache file can slow down the startup time.

By making the option to persist it to a file default to `False`, the object will be cached during a single usage of the pod (ex a `run` or `build`), but if the author of an extension or something internal wants it to be written to the file it is a conscious choice.

We can also highlight in the documentation for the object cache that if the author is persisting to the file they are also responsible for cleaning up unused cache items and that failing to do so can lead to bloat in the cache file.